### DISCUSSION TITLE
Discord server creation?

### DISCUSSION THREAD

**Discussion by Ethan-Hill** [vuejs/pinia#740] [Ideas]
Hey there,

I wanted to create this for people who are currently working with pinia and try to find out their thoughts on using a platform like discord to gather and discuss about this project and being able to support and help create ideas for the future. I know I'm a big believer in discord servers being a great place to collaborate and it would make the community's voices heard.

If people could add too this discussion on their thoughts on this that would be brilliant, thanks.

**Comment by posva** [comment]
The chat is the the same as Vuejs (Vue land). I don't think it's worth creating yet another server 😄

**Reply by d9k** [reply]
Confirm,  [Vuejs (Vue land) discord](https://discord.com/invite/vue) has [Pinia channel](https://discord.com/channels/325477692906536972/911637879212625971)
